batch/batch/batch.py

```diff
@@ -716,7 +794,7 @@ def create_job(jobs_builder, batch_id, userdata, job_spec):  # pylint: disable=R
         pvc_size = POD_VOLUME_SIZE
     always_run = job_spec.get('always_run', False)
 
-    pod_spec = batch_client.validate.job_spec_to_k8s_pod_spec(job_spec)
+    pod_spec = job_spec_to_k8s_pod_spec(job_spec)
```

Comment:
I think you can delete the batch_client import at the top of the file.

Reply:
Also, why do we still need the k8s pod spec?

Reply:
This is batch(1).  I didn't change it, it is still expecting a k8s pod spec.  I still need the batch_client to call the validation function in /jobs/create.